AvalonStudio/AvalonStudio.Languages.CSharp/MetaDataFile.cs

```diff
@@ -25,6 +25,8 @@ public MetaDataFile(IProject parentProject, Document document, string metaDataPa
             _document = document;
         }
 
+        public string ContentType => "CSharpMetadata";
```

Comment:
Currently when the code editor is provided with this, then no language service is found.

It should use the csharp language service as normal.